src/apiCalls.tsx

```diff
@@ -47,17 +47,15 @@ const checkResponse = (response: Response) => {
   return response.json()
 }
 
-export const getSentiment = (abstract: string) => {
-  return fetch(`https://api.dandelion.eu/datatxt/sent/v1/?lang=en&text=${abstract}&token=${
-    //Add your token here, without the template literal}
-  `)
+export const getSentiment = (abstract: string): Promise<number> => {
+  return fetch(`https://api.dandelion.eu/datatxt/sent/v1/?lang=en&text=${abstract}&token=91255d6d440f4c24a1b4a5ec443588d8`)
```

Comment:
Don't forget to remove your token, so we don't use up all your API fetch allowances 😆